yarn-project/prover-client/src/mocks/test_context.ts

```diff
@@ -37,7 +37,7 @@ import { getEnvironmentConfig, getSimulationProvider, makeGlobals } from './fixt
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +30 to +36]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Maybe for a future PR, I'm surprised all these are imported with relative paths and not with some @public/simulator, etc<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/10028?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->